standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveAlterHandler.java

```diff
@@ -803,6 +805,22 @@ public Partition alterPartition(RawStore msdb, Warehouse wh, String catName, Str
     return oldParts;
   }
 
+  private void checkTableTypeConversion(Database db, Table oldTbl, Table newTbl)
```

Comment:
Although the comment talks about external and managed the condition below just checks whether the table type has changed - an apparent disconnect between comment and code. The code is correct since we support only these two as table types right now. But either comment should explain this OR we should explicitly check tables type to be future proof.

Reply:
duplicate comment.